### ISSUE TITLE
Database issue

### ISSUE THREAD

**Issue by ACPK** [Gazler/ecto_admin_example_phoenix#1]
I had to remove the following from /config/config.exs

```
username: "user",
password: "pass",
```

**Comment by Gazler** [comment]
This is correct. This is just there for demonstrative purposes, it should be updated to match your environment. Perhaps the README should be updated to indicate this?
